kafka-ui-react-app/src/components/Dashboard/ClustersWidget/ClustersWidget.tsx

```diff
@@ -22,6 +23,10 @@ interface ChunkItem {
   data: Cluster[];
 }
 
+const SwitchWrapper = styled.div`
+  padding: 16px;
+`;
```

Comment:
It would be great if we can move this SwitchWrapper to the separate file `ClusterWidget.styled.ts`